x-pack/plugins/uptime/public/components/overview/monitor_list/troubleshoot_popover.tsx

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useState } from 'react';
+import {
+  EuiButton,
+  EuiButtonEmpty,
+  EuiText,
+  EuiPopoverTitle,
+  EuiPopover,
+  EuiPopoverFooter,
+} from '@elastic/eui';
+import { useSelector } from 'react-redux';
+import { FormattedMessage } from '@kbn/i18n-react';
+
+import { selectPingHistogram } from '../../../state/selectors';
+import { useUrlParams } from '../../../hooks';
+
+export const TroubleshootPopover = () => {
+  const [isPopoverOpen, setIsPopoverOpen] = useState(false);
+
+  const onButtonClick = () => setIsPopoverOpen((prevState) => !prevState);
+  const closePopover = () => setIsPopoverOpen(false);
+
+  const { pingHistogram } = useSelector(selectPingHistogram);
+
+  const updatedUrlParams = useUrlParams()[1];
+
+  const histogram = pingHistogram?.histogram ?? [];
+
+  const middleBucketTimestamp = histogram?.[Math.floor(histogram.length / 2)].x;
+  const firstBucketTimestamp = histogram?.[0].x;
+
+  return (
+    <EuiPopover
+      button={
+        <EuiButtonEmpty iconType="arrowDown" iconSide="right" onClick={onButtonClick}>
```

Comment:
remove icon from button. Could be confused as an accordion since it appears in the context of a table.